src/actions/index.js

```diff
@@ -51,6 +54,11 @@ export const downloadFileOffline = () => ({ type: DOWNLOAD_FILE_E_OFFLINE, alert
 
 export const openFolder = (folderId = ROOT_DIR_ID, isInitialFetch = false, router = null) => {
   return async dispatch => {
+    let routePrefix = '/files'
```

Comment:
I understand, but I'd suggest to add a comment here, because the code is not very explicit.